### ISSUE TITLE
Qwen3-Reranker模型调用的问题

### ISSUE THREAD

**Issue by czhcc** [QwenLM/Qwen3-Embedding#46]
按例子代码中process_inputs方法，会出现
You're using a Qwen2TokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.
/usr/local/lib/python3.12/dist-packages/transformers/tokenization_utils_base.py:2714: UserWarning: `max_length` is ignored when `padding`=`True` and there is no truncation strategy. To pad to max length, use `padding='max_length'`.
  warnings.warn(

信息。但如果修改代码
formatted_pairs = []
        for pair in pairs:
            # 手动添加prefix和suffix到文本中
            formatted_text = self.tokenizer.decode(self.prefix_tokens, skip_special_tokens=False) + pair + self.tokenizer.decode(self.suffix_tokens, skip_special_tokens=False)
            formatted_pairs.append(formatted_text)
        
        inputs = self.tokenizer(
            formatted_pairs,
            padding='max_length',  # 明确指定padding策略
            truncation=True,
            max_length=self.max_length,
            return_tensors="pt",
            return_attention_mask=True
        )

又会执行的特别慢。有什么办法吗？
